src/YieldDisburser.sol

```diff
@@ -247,33 +239,26 @@ contract YieldDisburser is OwnableUpgradeable {
 
         BREAD.claimYield(BREAD.yieldAccrued(), address(this));
         uint256 projectsLength = projects.length;
-        (uint256[] memory projectDistributions, uint256 totalVotes) = _commitVotedDistribution();
-        if (totalVotes == 0) {
-            projectDistributions = new uint256[](projectsLength);
-            for (uint256 i; i < projectsLength; ++i) {
-                projectDistributions[i] = 1;
-            }
-            totalVotes = projectsLength;
-        }
 
         lastClaimedTimestamp = Time.timestamp();
         lastClaimedBlockNumber = Time.blockNumber();
-        currentVotes = 0;
-
         uint256 halfBalance = BREAD.balanceOf(address(this)) / 2;
         uint256 baseSplit = halfBalance / projectsLength;
         uint256 percentageOfTotalVote;
         uint256 votedSplit;
         uint256[] memory votedSplits = new uint256[](projectsLength);
         uint256[] memory percentages = new uint256[](projectsLength);
         for (uint256 i; i < projectsLength; ++i) {
-            percentageOfTotalVote = projectDistributions[i] / totalVotes;
-            votedSplit = halfBalance * (projectDistributions[i] * PRECISION / totalVotes) / PRECISION;
+            percentageOfTotalVote = projectDistributions[i] / currentVotes;
```

Comment:
you will not get a percentage like this, you will also get zero if `currentVotes` is a summation of `projectDistributions` because `projectDistributions[i]` is necessarily smaller than `currentVotes` (unless they are equal which will never practically be the case)

Reply:
fixed with adab285